spec/features/admin/evaluations_spec.rb

```diff
@@ -10,38 +10,110 @@
   end
 
   describe 'Filters' do
+    let!(:user1) { create(:user,
+                          active: true,
+                          company: admin_user.company) }
+
+    let!(:user2) { create(:user,
+                          active: true,
+                          company: admin_user.company) }
```

Comment:
You could try to fit these into a single line, if it doesn't exceed the line limit.

You could also rename the variables to `evaluated_user` and `evaluator_user` for better readability.

You can also remove the `!` from the `let` declaration, since the users are already called when creating the evaluations, which are declared with a `!`.

Reply:
Thanks, I did it, and I didn't know about the differences between both lets, I will search for now.